orchestration/dagster_orchestration/hca_orchestration/tests/test_solids.py

```diff
@@ -74,14 +86,16 @@ def test_notifies_slack_with_failure_info(self):
                 run_config=self.solid_config,
                 mode_def=test_mode,
                 input_values={
-                    "validation_results": ProblemCount(duplicates=3, null_file_refs=2)
+                    "validation_results": ProblemCount(
+                        duplicates=3, null_file_refs=2, entities_with_dangling_project_refs=3
+                    )
                 }
             )
 
             expected_lines = [
                 "Problems identified in post-validation for HCA dev dataset fakedataset",
                 "Duplicate lines found: 3",
-                "Null file references found: 2",
+                "Null file references found: 2"
```

Comment:
leave my trailing commas alone :(

Reply:
that said, this line being changed makes me think we should update the slack notification solid to detect dangling project refs. currently, it'll only detect a validation as failed if it has either null file refs or duplicate rows.

Reply:
Fixed, and updated the slack notification.